shopping_mall/backend/app/routers/chatbot.py

```
@@ -477,3 +480,53 @@ def get_session_messages(
+
+
+# ===== Analytics Endpoints =====
+
+_PERIOD_DAYS = {"1d": 1, "7d": 7, "30d": 30}
+
+
+@router.get("/analytics/tools", response_model=ToolAnalyticsResponse)
+def get_tool_analytics(
+    period: str = Query("7d", description="집계 기간: 1d / 7d / 30d"),
+    tool_name: Optional[str] = Query(None, description="특정 도구 필터"),
+    authenticated_user_id: int = Depends(_get_current_user_id),
```

Ruff (0.15.10)

[warning] Unused function argument: `authenticated_user_id`

(ARG001)